packages/editor-ui/src/components/WorkflowHistory/WorkflowHistoryListItem.vue

```diff
@@ -0,0 +1,178 @@
+});
+
+const authors = computed<{ label: string; tooltip: string }>(() => {
+	const allAuthors = props.item.authors.split(', ');
+	let label = allAuthors[0];
+
+	if (allAuthors.length > 1) {
+		label = `${label} + ${allAuthors.length - 1}`;
+	}
+
+	return {
+		size: allAuthors.length,
+		label,
+	};
+});
+
+const idLabel = computed<string>(() =>
+	i18n.baseText('workflowHistory.item.id', { interpolate: { id: props.item.versionId } }),
+);
+
+const onAction = (action: TupleToUnion<WorkflowHistoryActionTypes>) => {
+	emit('action', { action, id: props.item.versionId });
+};
+
+const onVisibleChange = (visible: boolean) => {
+	actionsVisible.value = visible;
+};
+
+const onItemClick = (event: Event) => {
+	emit('preview', { event, id: props.item.versionId });
+};
+
+onMounted(() => {
+	emit('mounted', {
+		index: props.index,
+		offsetTop: itemElement.value?.offsetTop ?? 0,
+		active: props.active,
+	});
+});
+</script>
+<template>
+	<li
+		ref="itemElement"
+		:class="{
+			[$style.item]: true,
+			[$style.active]: props.active,
+			[$style.actionsVisible]: actionsVisible,
+		}"
+	>
+		<p @click="onItemClick">
+			<strong>{{ formattedCreatedAtDate }}</strong>
+			<n8n-tooltip placement="right-end" :disabled="authors.size < 2">
+				<template #content>{{ props.item.authors }}</template>
+				<span>{{ authors.label }}</span>
+			</n8n-tooltip>
+			<small>{{ idLabel }}</small>
+		</p>
+		<div :class="$style.tail">
+			<n8n-badge v-if="props.index === 0">
+				{{ i18n.baseText('workflowHistory.item.latest') }}
+			</n8n-badge>
+			<n8n-action-toggle
+				theme="dark"
+				:class="$style.actions"
+				:actions="props.actions"
+				@action="onAction"
+				@click.stop
+				@visible-change="onVisibleChange"
+				data-test-id="workflow-history-list-item-actions"
+			/>
+		</div>
+	</li>
+</template>
+<style module lang="scss">
+.item {
+	display: flex;
+	position: relative;
+	align-items: center;
+	justify-content: space-between;
+	border-left: var(--border-width-base) var(--border-style-base) var(--color-foreground-base);
+	border-bottom: var(--border-width-base) var(--border-style-base) var(--color-foreground-base);
+	color: var(--color-text-base);
+	font-size: var(--font-size-2xs);
+
+	p {
+		display: grid;
+		padding: var(--spacing-s);
+		line-height: unset;
+		cursor: pointer;
+
+		strong {
+			padding: 0 0 var(--spacing-2xs);
+			color: var(--color-text-dark);
+			font-size: var(--font-size-s);
+			font-weight: var(--font-weight-bold);
+		}
+
+		span {
+			justify-self: start;
+		}
+
+		small {
+			max-width: 160px;
+			white-space: nowrap;
+			overflow: hidden;
+			text-overflow: ellipsis;
+			padding: var(--spacing-4xs) 0 0;
+			font-size: var(--font-size-2xs);
+		}
+	}
+
+	.tail {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+	}
+
+	&.active {
+		background-color: var(--color-background-base);
+		border-left: 2px var(--border-style-base) var(--color-primary);
+
+		p {
+			padding-left: calc(var(--spacing-s) - 1px);
```

Comment:
The `1px` is the border?

Reply:
Yes, it didn't work with `calc(var(--spacing-s) - var(--border-style-base))` or maybe I mistyped something